test-cases/performance/perf-row-level-repair-1TB.yaml

```diff
@@ -0,0 +1,28 @@
+test_duration: 900
+
+prepare_write_cmd: ["cassandra-stress write no-warmup cl=QUORUM n=125000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..125000000",
+                    "cassandra-stress write no-warmup cl=QUORUM n=125000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=125000001..250000000",
+                    "cassandra-stress write no-warmup cl=QUORUM n=125000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=250000001..375000000",
+                    "cassandra-stress write no-warmup cl=QUORUM n=125000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=375000001..500000000",
+                    "cassandra-stress write no-warmup cl=QUORUM n=125000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=500000001..625000000",
+                    "cassandra-stress write no-warmup cl=QUORUM n=125000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=625000001..750000000",
+                    "cassandra-stress write no-warmup cl=QUORUM n=125000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=750000001..875000000",
+                    "cassandra-stress write no-warmup cl=QUORUM n=125000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=875000001..1000000000"]
+
+n_db_nodes: 3
+n_loaders: 8
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.4xlarge'
+instance_type_loader: 'c4.2xlarge'
+instance_type_monitor: 't3.small'
```

Comment:
why did you change default (`instance_type_monitor: 't3.large'`)? less than this type it won't work for you

Reply:
t3.small is used on all perf-regression yamls.

Reply:
remove monitor instance.